hw/top_earlgrey/data/ip/chip_rv_timer_testplan.hjson

```diff
@@ -0,0 +1,59 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: chip_rv_timer
+  testpoints: [
+    {
+      name: chip_sw_timer
+      desc: '''Verify the timeout interrupt assertion.
+
+            - Configure RV_TIMER to generate an interrupt after a set timeout.
+            - Issue a WFI to wait for the interrupt to trigger.
+            - Service the interrupt when it triggers; verify that it came from rv_timer.
+            - Verify that the interrupt triggered only after the timeout elapsed.
+            '''
+      stage: V2
+      si_stage: SV2
+      features: ["RV_TIMER.ENABLE", "RV_TIMER.CONFIG", "RV_TIMER.COMPARE", "RV_TIMER.INTERRUPT",
+                 "RV_TIMER.COUNTER"]
+      tests: ["chip_sw_rv_timer_irq"]
+    }
+    {
+      name: tick_configuration
+      desc: '''Verify that the timer can be configured to generate a system tick.
+
+            - Configure the timer to generate a tick of `T` microseconds long.
```

Comment:
This will test a single ratio of step and pre_scaler, you may want to repeat for values not multiples of 10.

Reply:
done.